### PR TITLE
Use justbytes lib to print values with units

### LINKED ISSUE #74: Sizes should be expressed in human units
e.g. in "stratis pool list", using a function like:
```
def sizeof_fmt(num, suffix='B'):
    for unit in ['','Ki','Mi','Gi','Ti','Pi','Ei','Zi']:
        if abs(num) < 1024.0:
            return "%3.1f%s%s" % (num, unit, suffix)
        num /= 1024.0
    return "%.1f%s%s" % (num, 'Yi', suffix)
```
(from https://stackoverflow.com/questions/1094841/reusable-library-to-get-human-readable-version-of-file-size)

### PR DESCRIPTION
**Pull request by agrover**
We only do this in one spot now but will likely be printing such values
much more in the future.

fixes #74

Signed-off-by: Andy Grover <agrover@redhat.com>

### CODE REVIEW

File: src/stratis_cli/_actions/_top.py [hunk 1]
```diff
@@ -75,8 +78,8 @@ def list_pools(_):
         tables = [
            [
               mopool.Name(),
-              mopool.TotalPhysicalSize(),
-              mopool.TotalPhysicalUsed()
+              str(Range(mopool.TotalPhysicalSize(), SECTOR_SIZE)),
+              str(Range(mopool.TotalPhysicalUsed(), SECTOR_SIZE)),
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_top.py]: You need to fix heading strings, below.

File: src/stratis_cli/_actions/_top.py [hunk 2]
```diff
@@ -75,8 +78,8 @@ def list_pools(_):
         tables = [
            [
               mopool.Name(),
-              mopool.TotalPhysicalSize(),
-              mopool.TotalPhysicalUsed()
+              str(Range(mopool.TotalPhysicalSize(), SECTOR_SIZE)),
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_top.py]: Note that you're accepting the default configuration here. The defaults were chosen by me as the most useful for humans representation I could come up with. But I'm not everybody's kind of human.

Comment (agrover) [reply src/stratis_cli/_actions/_top.py]: heh. Well it looked ok to me!

File: src/stratis_cli/_actions/_top.py (hunk 1 above)
Comment (agrover) [reply src/stratis_cli/_actions/_top.py]: ok.

**Comment by agrover** [comment]
made changes.

**Comment by agrover** [comment]
ok, justbytes>=0.10 in setup.py.

**Pull request merged**